parm/config/config.defaults.s2sw

```diff
@@ -16,13 +16,13 @@ use_cice_alb=".true."
 
 # config.wave
 
-waveGRD='mx025'
-waveinterpGRD='reg025'
-waveuoutpGRD='mx025'
-MESH_WAV='mesh.mx025.nc'
+waveGRD='gefsv13_025'
```

Comment:
Could you explain what is the difference between the grids?
Both are "quarter-degree" I assume.
Adding application specific names has got us into trouble before, so I would like to know more what the actual differences are then propose a solution that will work across the GFS and GEFS infrastructures.

Reply:
Jessica explained the difference between mx and gefsv13 above.

Reply:
Apologies, I did not refresh by browser to see comments from the previous reviews.
Instead of `gefsv13_`, I would propose something more generic.  If `mx` stands for "tripolar" what would an appropriate prefix for "regular" be?